src/hotspot/cpu/riscv/interp_masm_riscv.cpp

```diff
@@ -179,15 +179,10 @@ void InterpreterMacroAssembler::check_and_handle_earlyret(Register java_thread)
 
 void InterpreterMacroAssembler::get_unsigned_2_byte_index_at_bcp(Register reg, int bcp_offset) {
   assert(bcp_offset >= 0, "bcp is still pointing to start of bytecode");
-  if (AvoidUnalignedAccesses && (bcp_offset % 2)) {
-    lbu(t1, Address(xbcp, bcp_offset));
-    lbu(reg, Address(xbcp, bcp_offset + 1));
-    slli(t1, t1, 8);
-    add(reg, reg, t1);
-  } else {
-    lhu(reg, Address(xbcp, bcp_offset));
-    revb_h_h_u(reg, reg);
-  }
+  lbu(t1, Address(xbcp, bcp_offset));
```

Comment:
maybe we could expand the revb_h_h here? something like below:
```
if ((AvoidUnalignedAccesses && (bcp_offset % 2)) || !UseZbb) {
  ...
} else {
  // expand revb_h_h_u when +UseZbb
}
```

Reply:
similar comments for below changes.

Reply:
Ah, that looks more complex then before from a maintainance perspective. We only get one instruction saved which is not on the performance critical path. I guess it doesn't worth it?

Reply:
Yes, I have the similar concerns. :)

Reply:
Yeah! Also I don't think it's a good idea to spread uses of extension options like `UseZbb` everywhere. It will be more reasonable to me to encapsulate it in some assember routines which are more generic to the callers.